tests/integration/test_store.py

```diff
@@ -220,7 +220,7 @@ def test_store_buffer_size(mini_sentry, relay):
             relay.send_event(project_id, {"message": "pls ignore"})
         pytest.raises(queue.Empty, lambda: mini_sentry.captured_events.get(timeout=1))
 
-        for (_, error) in mini_sentry.test_failures:
+        for _, error in mini_sentry.test_failures:
```

Comment:
I didn't want to do this change, but it happens automatically every time i press save

Reply:
`git checkout master tests/integration/test_store.py`